pkgs/development/python-modules/bootstrapped-pytest/default.nix

```diff
@@ -0,0 +1,147 @@
+# A pytest(-xdist) package that does not depend on any other python packages
+# Can be used to test all other packages without causing infinite recursion
+
+{ bootstrapped-pip
+, fetchPypi
+, isPy27
+, lib
+, makeWrapper
+, python
+, stdenv
+, unzip
+
+# python packages to take sources from
```

Comment:
Seems like we should have an (internal) function for building/installing an environment using pip.